scraper.py

```diff
@@ -158,5 +155,5 @@ def scrape_book_for_category(category):
     categorieses = get_categories_from_side_bar()
     create_folder(target_folder)
```

Comment:
why are you creating the folder there. I thought we would have one folder per category?